plugin/adsk/scripts/mayaUsdTranslatorImport.mel

```diff
@@ -229,24 +229,26 @@ global proc int mayaUsdTranslatorImport (string $parent,
 //            menuItem -label "Object space";
 //            menuItem -label "World space";
 //        optionMenuGrp -e -sl 1 mayaUsdTranslator_CoordSystemOptionMenu;
+        frameLayout -label `getMayaUsdString("kImportMaterialsLbl")` materialsFrameLayout;
+            separator -style "none";
+            checkBoxGrp -label "" -label1 `getMayaUsdString("kImportMaterialsLbl")` -cw 1 $cw1 -value1 1 -ann `getMayaUsdString("kImportMaterialsAnn")` -cc ("mayaUsdTranslatorImport_MaterialsCB") mayaUsdTranslator_MaterialsCheckBox;
```

Comment:
I'm not sure what the default is, but I think you should pass the number of checkboxes with `-ncb 1` and I don't think you need `-label ""` as that flag is optional. Same with all the ones below.

Reply:
Hi @seando-adsk , `-label ""` is actually needed to achieve the alignment. Without `-label` the option will be aligned to the left as shown below: 
![image](https://github.com/Autodesk/maya-usd/assets/127771016/380b315c-0515-4514-b2b1-c9ae18e9b215)

For `-ncb` I'm not sure it's necessary.. We didn't use that option before so I think that's more for the case that we have multiple checkboxes?

Reply:
Okay for the empty label - I didn't realize that.
I checked the code for the checkBoxGrp command and it does indeed default to 1.